src/components/Wrapper.js

```diff
@@ -33,7 +34,8 @@ export const InfoBox = styled(WrapperRow)`
   margin-bottom: ${normalize(5)}px;
 `;
 
-export const WrapperWithOrientation = memo(({ orientation, dimensions, children }) => {
+export const WrapperWithOrientation = ({children}) => {
+  const {orientation, dimensions} = useContext(OrientationContext);
```

Comment:
nice! very obvious to do it this way.

Reply:
but i see something not changed on landscape. when i log, there is no re-rendering when i turn the phone. this needs some investigation. it does not work the same way as in bad belzig app in testflight for example.
when i turn the phone inside an POI, the layout does not change.

Reply:
there is an underlying issue with the orientation context.

the orientation listener is either unregistered somewhere, or stops working entirely under android.
this happens in master as well.

Reply:
i see the correct behaviour for android in master and for example live Bad Belzig App. we can talk about it in our call.

Reply:
added a workaround for the size of the map in 6ad5fba4f34d194fe49bc6f978ee95c95cb9b373

Reply:
as discussed there is a react-native 0.63 bug that causes the orientation update to get lost on android (refer [here](https://github.com/facebook/react-native/issues/29451))

that means that this is the best we can do with our limited time